.release/update_versions.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# Copyright 2021 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+VERSION=$(cat version.txt)
+# Update parent pom version
+mvn versions:set -DnewVersion=$VERSION -DgenerateBackupPoms=false
+
+# Update revision property which is used to determine core dependency version
+mvn versions:set-property -Dproperty="revision" -DnewVersion=$VERSION -DgenerateBackupPoms=false
```

Comment:
I think maybe I didn't explain myself correctly. The link I had before mentioned that if we don't specify a version for the child pom, it'll inherit the version from the parent pom. This means we don't need to use `revision` or pom flattening, and `mvn versions:set` will update the parent version in all the poms.

Reply:
So here we're using `revision` to substitute the version for where we add the core socket factory as a dependency and specify the version. Not sure if that one can be inherited.

Reply:
Totally missed that - in that case, can we just just add it to `dependencyManagement` in the parent pom, with `${project.version}`?

Reply:
I think so